executor/executor_test.go

```diff
@@ -98,9 +99,11 @@ var _ = Suite(&testOOMSuite{})
 var _ = Suite(&testPointGetSuite{})
 var _ = Suite(&testRecoverTable{})
 var _ = Suite(&testFlushSuite{})
-var _ = Suite(&testShowStatsSuite{})
 
-type testSuite struct {
+type testSuite struct{ *testSuiteChild }
+type testSuiteP1 struct{ *testSuiteChild }
+
+type testSuiteChild struct {
```

Comment:
Could you please improve the naming here? e.g, `testSuiteChild` -> `baseTestSuite`, and so on?

Reply:
Done.